open_in_mpv/open_in_mpv.py

```diff
@@ -15,33 +14,85 @@
 import click
 import xdg.BaseDirectory
 
+from .constants import IS_MAC, IS_WIN, MAC_BIN_PATH
+
+FALLBACKS: Final[dict[str, Any]] = {
+    'log': None,
+    'socket': None
+}
 
 @lru_cache()
 def get_log_path() -> str:
-    if platform.mac_ver()[0]:
+    if IS_MAC:
         return expanduser('~/Library/Logs')
+    if IS_WIN:
+        return expandvars(r'%LOCALDATA%\open-in-mpv') # cspell:disable-line
     try:
         return xdg.BaseDirectory.save_state_path('open-in-mpv')
     except KeyError:
-        with tempfile.TemporaryDirectory(prefix='open-in-mpv') as dir_:
-            return dir_
-
+        FALLBACKS['log'] = tempfile.TemporaryDirectory(prefix='open-in-mpv') # pylint: disable=R1732
+        return str(FALLBACKS['log'].name)
 
```

Comment:
There should be two lines before every function definition. The formatter should take care of this (`yarn format` to make this easier).